APISpecification.yaml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.0
+info:
+  title: Tools api
+  version: 0.0.1
+paths:
+  /api/tools/vep/submissions/{id}:
+    get:
+      description: Returns results of VEP analysis
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: page
+          in: query
+          required: false
+          schema:
+            type: number
+            default: 1
+        - name: per_page
+          in: query
+          required: false
+          schema:
+            type: number
+            default: 100
+      responses:
+        '200':
+          description: Successful operaiton.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VepResultsResponse'
+components:
+  schemas:
+    PaginationMetadata:
+      type: object
+      required:
+        - page
+        - per_page
+        - total
+      properties:
+        page:
+          type: number
```

Comment:
Not absolutely necessary but adding example here would populate the example in swagger with real values instead of the type.